econpizza/__init__.py

```diff
@@ -14,6 +14,7 @@
 from .solvers.solve_linear_state_space import solve_linear_state_space, find_path_linear_state_space
 from .solvers.shooting import find_path_shooting
 from .parser import parse, load
+from .config import ECONPIZZA_CACHE_DIR, JAX_CACHE_DIR, config
```

Comment:
For the cache dirs we should use the default that python is using for `__pycache__` and compiled numba functions.

Reply:
On this point, do you mean we have to create the caches inside the end user working directory? This would mean to inspect the context in which the function is called.
https://stackoverflow.com/questions/37546685/get-absolute-path-of-caller-file

It's possible, I'll try it out. 
I was thinking whether there would be a case where the folder will not be writable and this would fail, maybe it's an edge case which will never happen.

In this case I think we won't need the temp files library all files will go into these folders inside the user's working directory and they can be deleted all together(reset).

Reply:
Is this what `__pycache__` is always doing?

Reply:
I would guess so, yes. The config code from `econpizza` will have to know the place from which it was called in order to create a folder there.

Reply:
Please check what the default for `.pyc` and numba is and use the same structure.